onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -973,7 +970,6 @@ def aten_bernoulli_p(self: TTensor, p: float) -> TTensor:
 
     Ignore `generator` due to the limit on ONNX expressiveness.
     """
-    # NOTE: We will lose some precision when input is float64 but that's considered insignificant
```

Comment:
is this deletion intended?

Reply:
Yes, because we don't rely on self's value to create the result

Reply:
I will add it back.